### DISCUSSION TITLE
I can't install it.

### DISCUSSION THREAD

**Discussion by Tubelerczyk** [HerMajestyDrMona/Windows11DragAndDropToTaskbarFix#102] [Q&A]
I can't install it in Windows 11, no version of the program. When I click on the file it loads, but then nothing happens.

**Comment by micheldelsent** [comment]
Execute como administrador, ele vai direto pra barra de tarefas e fica executável - nada mais é necessário, talvez, nas 'configurações do txt' precise colocar 1 para iniciar como admin. 
![Screenshot_6](https://github.com/user-attachments/assets/42cb68d8-2483-4e4b-8a91-eac1ef70ca41)
